data/mods/Magiclysm/bionics.json

```diff
@@ -5,7 +5,7 @@
     "name": { "str": "Blood Power Generator CBM" },
     "description": "Embedded into your back and connected to your bionic power supply is a powerfull spell focus able to consume blood from dead creatures to produce bionic power.  It can store up to 100 mL of blood.",
     "occupied_bodyparts": [ [ "torso", 8 ] ],
-    "fuel_options": [ "blood", "dragon_blood", "tainted_blood" ],
+    "fuel_options": [ "blood", "dragon_blood", "tainted_blood", "hblood" ],
```

Comment:
Shouldn't "tainted_blood" here be replaced with "blood_tainted" as is set elsewhere in this PR?

Reply:
well drat. that means we have to make another PR.